OpenLIFUSonicationControl/OpenLIFUSonicationControl.py

```diff
@@ -150,6 +154,15 @@ def setParameterNode(self, inputParameterNode: Optional[OpenLIFUSonicationContro
             # Note: in the .ui file, a Qt dynamic property called "SlicerParameterName" is set on each
             # ui element that needs connection.
             self._parameterNodeGuiTag = self._parameterNode.connectGui(self.ui)
+
+    def onRunClicked(self):
+        print("Run sonication control")
```

Comment:
it should not say "control"

maybe also would be better to say "placeholder" somehwere in the printed text